modules/objdetect/include/opencv2/objdetect.hpp

```diff
@@ -304,7 +305,8 @@ class CV_EXPORTS_W CascadeClassifier
                                   int minNeighbors = 3, int flags = 0,
                                   Size minSize = Size(),
                                   Size maxSize = Size(),
-                                  bool outputRejectLevels = false );
+                                  bool outputRejectLevels = false,
+                                  int rejectStage = 3);
```

Comment:
yeah but that behaviour was bogus .. it is not natural... so I would not do that.